app.js

```diff
@@ -40,7 +40,7 @@ app.use(require('express-session')({
 app.use(passport.initialize());
 app.use(passport.session());
 
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(path.join(__dirname, 'public')));;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary semicolon.](https://app.codacy.com/manual/shushwenira/Volunteer_Sign_Up/pullRequest?prid=4945039)

Reply:
Fix the issue